_blog.yml

```
@@ -1874,3 +1874,12 @@
   tags:
     - ethics
     - diffusers
+
+
+- local: controlnet
+  title: "Welcoming ControlNet to Diffusers 🧨"
+  author: patrickvonplaten
```

```suggestion
  author: patrickvonplaten
```

Please add yourself here, I didn't write anything